src/index.ts

```
@@ -1,162 +1,265 @@
-  public static getInstance (): CreatorsCloudStorageClient {
+  public static getInstance(): CreatorsCloudStorageClient {
     if (!CreatorsCloudStorageClient.instance) {
       throw new Error(`${CreatorsCloudStorageClient.name} not initialized`)
     }
 
     return CreatorsCloudStorageClient.instance
   }
 
-  public static init (region: string, accessKeyId: string, secretAccessKey: string, loggerInstance: Logger, errorConverter: ErrorConverter): void {
+  public static init(region: string, accessKeyId: string, secretAccessKey: string, loggerInstance: Logger, errorConverter: ErrorConverter): void {
     this.instance = new CreatorsCloudStorageClient(region, accessKeyId, secretAccessKey, loggerInstance, errorConverter)
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)